src/styles/blocks/card.scss

```
@@ -0,0 +1,136 @@
+// use nprmalize scss
+body {
+  margin: 0;
+  padding: 0;
+
+  font-weight: 400;
+  line-height: 14px;
+}
+
+h3 {
+  font-weight: 500;
+}
+
+.card {
+  box-sizing: border-box;
+
+  height: 408px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  background-color: $primary-bg-color;
+  border: 1px solid $primary-bg-color;
+
+  &__image{
+    margin: 0;
+    padding: 32px 20px 0;
+    height: 134px;
+    width: 160px;
+
+    object-fit: contain;
+    object-position: center;
+    align-self: center;
+  }
+
+  &__product-name {
+    margin: 0;
```

use nesting here, also, check another places in your code
```suggestion

  &__product {
    &-name {
      margin: 0;
```